### PR TITLE
Rsc7b

### PR DESCRIPTION
**Pull request by EvgenyKarkan**

### CODE REVIEW

File: Spec/RestClient.swift [hunk 1]
```diff
@@ -832,7 +832,22 @@ class RestClient: QuickSpec {
...
+            it("X-Ably-Lib: [lib][.optional variant]?-[version] should be included in all REST requests") {
+                let client = ARTRest(options: AblyTests.commonAppSetup())
+                client.httpExecutor = testHTTPExecutor
+                let channel = client.channels.get("test")
+                waitUntil(timeout: testTimeout) { done in
+                    channel.publish(nil, data: "message") { error in
+                        expect(error).to(beNil())
+                        let headerLibVersion = testHTTPExecutor.requests.first!.allHTTPHeaderFields?["X-Ably-Lib"]
+                        let ablyBundleLibVersion = ARTDefault.libraryVersion()
+                        expect(headerLibVersion).to(equal(ablyBundleLibVersion))
```
Comment (mattheworiordan) [inline Spec/RestClient.swift]: Whilst I understand that you don't necessarily want the the test to change for every time a version changes, it does feel like this test needs to be a bit more verbose i.e. can it not at least check that the header matches `ios-0.8.*` for example?  It just means the test confirms what we expect as opposed to simply referencing another part of the internals of the lib.

**Comment by mattheworiordan** [comment]
Thansk, minor comments otherwise good, 👍

**Comment by EvgenyKarkan** [comment]
Thanks for feedback, I'll try to improve it.

**Comment by EvgenyKarkan** [comment]
Hi @mattheworiordan, I've added commit, please find time to review it.

**Comment by mattheworiordan** [comment]
Thanks, 👍

**Comment by mattheworiordan** [comment]
Can this PR please be rebased onto master as I cannot merge automatically?

**Comment by EvgenyKarkan** [comment]
Morning @mattheworiordan , no more conflicts here.

**Comment by mattheworiordan** [comment]
Thanks @EvgenyKarkan

**Pull request merged**